test/apps/iodemo/ucx_wrapper.h

```diff
@@ -398,6 +407,7 @@ class UcxConnection {
     uint64_t        _conn_id;
     uint64_t        _remote_conn_id;
     char            _log_prefix[MAX_LOG_PREFIX_SIZE];
+    char            _conn_log_prefix[MAX_LOG_PREFIX_SIZE];
```

Comment:
should not we have only one logging format initialized according to parameter?

Reply:
The added format contains 4 tuples (local gid,lid <-> remote gid,lid) to identify a connection and used when per-connection info is needed. The original format remains used in normal case. Adding that 4-tuple identifier to the original format looks too lengthy.

Reply:
I mean reuse `_log_prefix` initialized depending on `test_opts->per_conn_info` value